src/game/option_passport.c

```diff
@@ -205,9 +247,114 @@ static void Option_PassportInitSaveRequester(void)
         req->vis_lines = 12;
     }
 
+    Savegame_ScanAvailableLevels(req);
+}
+
+static void Option_PassportInitSaveRequester(int16_t page_num)
+{
+    REQUEST_INFO *req = &g_LoadSavegameRequester;
+    InitRequester(req);
+    if (page_num == PASSPORT_PAGE_1) {
+        SetRequesterHeading(req, g_GameFlow.strings[GS_PASSPORT_LOAD_GAME]);
+    } else {
+        SetRequesterHeading(req, g_GameFlow.strings[GS_PASSPORT_SAVE_GAME]);
+    }
```

Comment:
This change renames the Load and Save requestor header to Load Game or Save Game depending on which passport page it's called from. Select Level was the old name and is now properly moved to the new Select Level menu.

Reply:
Yeah, good